packages/sdk-core/src/model/configuration/BacktraceDatabaseConfiguration.ts

```diff
@@ -43,6 +43,12 @@ export interface EnabledBacktraceDatabaseConfiguration {
      * A crash report is generated, stored locally, and uploaded upon next start.
      */
     captureNativeCrashes?: boolean;
+
+    /**
+     * Controls how much previous session caches to preserve before sending data from previous sessions.
```

Comment:
Maybe it's worth to mention this option only applies for session files and is not applied to database records/generated reports in the database?

Reply:
Good idea, changed